src/accounts/validators.py

```diff
@@ -0,0 +1,47 @@
+import re
+from django.core.exceptions import ValidationError
+from django.core.validators import URLValidator
+
+
+def validate_year(value):
+    reg = re.compile(r'[1-5]')
+    if not reg.match(value):
+        raise ValidationError('Please enter a value between 1-5')
+    return value
+
+
+def validate_gpa(value):
+    reg = re.compile(r'^\d\.?\d*$')
+    if not reg.match(value):
+        raise ValidationError('This field should contain only numeric values')
+    return value
+
+
+def validate_phone(value):
```

Comment:
We should remove this. As we other care in what format they enter the phone no. Like I can begin my phone number with 0 if I want to and have an 11 digit number